src/it/override-war-with-snapshot/war-with-plugin/pom.xml

```
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>4.61</version>
+    <relativePath />
+  </parent>
+
+  <groupId>org.jenkins-ci.tools.hpi.its</groupId>
+  <artifactId>override-war-with-snapshot</artifactId>
+  <version>${revision}</version>
+  <packaging>hpi</packaging>
+  <name>MyNewPlugin</name>
+  <description>My New Plugin</description>
+  <url>https://github.com/jenkinsci/verify-it-plugin</url>
+
+  <licenses>
+    <license>
+      <name>MIT License</name>
+      <url>https://opensource.org/licenses/MIT</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>Bob</id>
+      <name>Bob McBobfaceNoam Chomsky</name>
```

Multiple names when a single name is expected

```suggestion
      <name>Bob McBobface</name>
```